src/Aspire.Hosting/Dashboard/DashboardOptions.cs

```diff
@@ -31,6 +32,10 @@ public void Configure(DashboardOptions options)
         options.OtlpApiKey = configuration["AppHost:OtlpApiKey"];
 
         options.AspNetCoreEnvironment = configuration["ASPNETCORE_ENVIRONMENT"] ?? "Production";
+
+        options.TelemetryEnabled = bool.TryParse(configuration["DOTNET_DASHBOARD_ENABLE_TELEMETRY"], out var telemetryEnabled)
```

Comment:
Rename this to `DOTNET_DASHBOARD_TELEMETRY_OPTOUT` and invert the logic.

Reply:
I know we already have "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL" and "DOTNET_DASHBOARD_OTLP_HTTP_ENDPOINT_URL" but feels like it ought to be "ASPIRE_DASHBOARD_TELEMETRY_OPTOUT". No need to be unnecessarily dotnet centric even more than already. @DamianEdwards thoughts?

Reply:
For future env vars, we should probably use ASPIRE_ not DOTNET_ and for existing DOTNET_ ones should possibly also accept and document ASPIRE_ ... but that's a problem for later

Reply:
Agree with @JamesNK

Reply:
On naming, I agree also with @danmoseley that we should move to using `ASPIRE_` as our prefix (with back compat for the existing ones of course). We should do that all up in 9.2 I think.

Reply:
Opened https://github.com/dotnet/aspire/issues/8009